### PR TITLE
Add Scraper for Google Civics API

### LINKED ISSUE #2: Connect to Civic Information API

### LINKED ISSUE #5: create classes to represent API JSON objects

### LINKED ISSUE #21: Store email field from API JSON
Example in Texas ltgov

Related to #2 
Related to #5

### PR DESCRIPTION
**Pull request by tirtow**
This PR adds in `OfficialScraper` to use to scrape information off the [Google Civics Information API](https://developers.google.com/civic-information/docs/v2/representatives/representativeInfoByAddress) into `Official` objects.

closes #2 
closes #5 
closes #21

### CODE REVIEW

File: RepresentsMe.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -7,6 +7,16 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		13287BB92220479B00622CD4 /* OfficialScraperTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 13287BB82220479B00622CD4 /* OfficialScraperTests.swift */; };
+		13287BBB2220490E00622CD4 /* apikey.swift in Sources */ = {isa = PBXBuildFile; fileRef = 13287BBA2220490E00622CD4 /* apikey.swift */; };
```
Comment (bzsinger) [inline RepresentsMe.xcodeproj/project.pbxproj]: Yeah this is what I'm worried about

Comment (bzsinger) [reply RepresentsMe.xcodeproj/project.pbxproj]: Deemed good based on offline conversation. @tirtow will move apikey.swift to main folder and organize API scraping files into folder(s)

**Review by bzsinger — CHANGES_REQUESTED** [review]
See comment.

File: RepresentsMe.xcodeproj/project.pbxproj (hunk 1 above)
Comment (tirtow) [reply RepresentsMe.xcodeproj/project.pbxproj]: 86e1d35 resolves this.

**Review by bzsinger — APPROVED** [review]

**Pull request merged**